test.suml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Classdiagram><settings><option name="paintImplements" value="true" /><option name="paintDepends" value="false" /><option name="layoutOnChanges" value="false" /><option name="paintUses" value="false" /><option name="paintInner" value="true" /><option name="paintExtends" value="false" /></settings><classes /><stickycomponents /><textcomponents /><connectors /></Classdiagram>

```

Comment:
What is this?
If you know, just ignore this comment

Reply:
It's from SimpleUml. I thought I deleted it!